lambda-sqs-terraform/main.tf

```diff
@@ -64,11 +60,21 @@ resource "aws_iam_role" "lambda_iam_role" {
 EOF
 }
 
+resource "aws_iam_role_policy_attachment" "lambda_basic_execution" {
+  role       = aws_iam_role.lambda_iam_role.name
+  policy_arn = data.aws_iam_policy.lambda_basic_execution_role_policy.arn
+}
+
+resource "aws_iam_role_policy_attachment" "lambda_sqs" {
+  role       = aws_iam_role.lambda_iam_role.name
+  policy_arn = aws_iam_policy.lambda_policy.arn
+}
+
```

Comment:
note: The `managed_policy_arns` argument is already deprecated. I update code correctly😀 See document below.

- https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role